hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/create/CreateDecoder.java

```diff
@@ -53,6 +55,7 @@ public class CreateDecoder {
     private static final int FRACTIONAL_FEE = 4;
     private static final int NFT_FIXED_FEE = 1;
     private static final int NFT_ROYALTY_FEE = 2;
+    private static final int METADATA = 9;
```

Comment:
nit: we should give these more descriptive names. 
In use I wouldn't know what this referred to except for coming back and reading the comment a few lines up.
Maybe
```suggestion
    private static final int TOKEN_METADATA_TUPLE_INDEX = 9;
```
or
```suggestion
    private static final int TOKEN_METADATA_INDEX = 9;
```
or something better

Reply:
We have a comment explaining the constants here and imo it makes it more readable when you use it -> `call.get(METADATA);` to `call.get(TOKEN_METADATA_INDEX);`

Reply:
nit: I don' think the comment is descriptive enough if we're relying on that.
If doesn't convey the logic or assumption of the tuple and index relevance. There's a lot of presume knowledge